### ISSUE TITLE
unescape link when copied

### ISSUE THREAD

**Issue by exhesham** [reduckted/GitWebLinks#16]
the copied link replace '/' with %2f and the browser fails to open it:
You should not encode these:
```
            encodeURI(branchOrHash),
            encodeURI(relativePathToFile)
```
at
https://github.com/reduckted/vscode-gitweblinks/blob/529a64b28f3b9675a0cf150a30034b448413702f/src/links/BitbucketCloudHandler.ts#L41

**Comment by reduckted** [comment]
Hi, thanks for reporting this. It looks like Bitbucket Cloud has changed their URL formats since I first wrote this extension, because there are a few other differences in the URLs that I've noticed as well. I'm in the middle of a major rewrite of this extension, so I'll fix this as part of the changes I'm making. I can't promise anything, but hopefully I'll have it finished in about a week.

**Comment by reduckted** [comment]
Hi @exhesham, I've just published v2.0.0 on the marketplace. I've tried a few different bitbucket.org repos and the links are generated correctly for me. Let me know if you're still getting incorrect links generated with the new version.

**Comment by exhesham** [comment]
Hi,
I tested the updated version on bitbucket and it works like a charm

thank you very much.
